src/ProjectTemplates/scripts/Run-Blazor-Locally.ps1

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env pwsh
+#requires -version 4
+
+[CmdletBinding(PositionalBinding = $false)]
+param()
+
+Set-StrictMode -Version 2
+$ErrorActionPreference = 'Stop'
+
+. $PSScriptRoot\Test-Template.ps1
+
+Test-Template "blazorserverside" "blazorserverside" "Microsoft.DotNet.Web.ProjectTemplates.3.0.3.0.0-dev.nupkg" $false
```

Comment:
Did you mean to check this in? If so can you add a comment at the top to say what this does and when to use it?